test/input.txt

```diff
@@ -148,6 +148,18 @@
   clear
   list
 
+##########################################################
+# test sort command
+##########################################################
+
+  # add some contacts into address book
```

Comment:
Good job in adding a test case for the IO, maybe you can start trying out the JUnit test suite to familiarise yourself with it.